src/ml-service/packages/sensor_location/app/modules/locate_model.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
```

Comment:
When we starting refactoring our codebase, the connection details of our databases will easily fall under` AirQo-api/src/locate/config/db_connection`.
This will ensure that we always call our database connection once throughout our codebase.